src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -182,15 +191,17 @@
   [oid-map :- OIDMap
    ace :- ACE
    extensions :- Extensions]
-  (let [match-key (fn [k]
+  (let [oid-map (merge default-oid-map oid-map)
```

Comment:
As a minor style thing, I'd tend to prefer using a different name than `oid-map` for the bound variable - even if just `oid-map'`.  Same for the redefined `ace-value` variable below.